--- examples/Makefile.orig	2015-07-23 19:08:32.000000000 +0200
+++ examples/Makefile	2016-03-20 23:44:01.052827911 +0100
@@ -20,28 +20,28 @@
 
 include $(topdir)/Makefile.inc
 
-LIBMUMPS_COMMON = $(libdir)/libmumps_common$(PLAT)$(LIBEXT)
+LIBMUMPS_COMMON = $(libdir)/libmumpso_common$(PLAT)$(LIBEXT)
 
 
-LIBSMUMPS = $(libdir)/libsmumps$(PLAT)$(LIBEXT) $(LIBMUMPS_COMMON)
+LIBSMUMPS = $(libdir)/libsmumpso$(PLAT)$(LIBEXT) $(LIBMUMPS_COMMON)
 
 ssimpletest:  $(LIBSMUMPS)  $$@.o
 	$(FL) -o $@ $(OPTL) ssimpletest.o  $(LIBSMUMPS) $(LORDERINGS) $(LIBS) $(MUMPS_LIBF77) $(LIBBLAS) $(LIBOTHERS)
 
 
-LIBDMUMPS = $(libdir)/libdmumps$(PLAT)$(LIBEXT) $(LIBMUMPS_COMMON)
+LIBDMUMPS = $(libdir)/libdmumpso$(PLAT)$(LIBEXT) $(LIBMUMPS_COMMON)
 
 dsimpletest: $(LIBDMUMPS)  $$@.o 
 	$(FL) -o $@ $(OPTL) dsimpletest.o  $(LIBDMUMPS) $(LORDERINGS) $(LIBS) $(MUMPS_LIBF77) $(LIBBLAS) $(LIBOTHERS)
 
 
-LIBCMUMPS = $(libdir)/libcmumps$(PLAT)$(LIBEXT) $(LIBMUMPS_COMMON)
+LIBCMUMPS = $(libdir)/libcmumpso$(PLAT)$(LIBEXT) $(LIBMUMPS_COMMON)
 
 csimpletest: $(LIBCMUMPS)  $$@.o
 	$(FL) -o $@ $(OPTL) csimpletest.o  $(LIBCMUMPS) $(LORDERINGS) $(LIBS) $(MUMPS_LIBF77) $(LIBBLAS) $(LIBOTHERS)
 
 
-LIBZMUMPS = $(libdir)/libzmumps$(PLAT)$(LIBEXT) $(LIBMUMPS_COMMON)
+LIBZMUMPS = $(libdir)/libzmumpso$(PLAT)$(LIBEXT) $(LIBMUMPS_COMMON)
 
 zsimpletest: $(LIBZMUMPS)  $$@.o
 	$(FL) -o $@ $(OPTL) zsimpletest.o  $(LIBZMUMPS) $(LORDERINGS) $(LIBS) $(MUMPS_LIBF77) $(LIBBLAS) $(LIBOTHERS)
@@ -58,19 +58,19 @@
 	$(CC) $(OPTC) $(INCS) $(CDEFS) -I. -I$(topdir)/include -I$(topdir)/src -c $*.c $(OUTC)$*.o
 
 
-$(libdir)/libsmumps$(PLAT)$(LIBEXT):
+$(libdir)/libsmumpso$(PLAT)$(LIBEXT):
 	@echo 'Error: you should build the library' $@ 'first'
 	exit 1
 
-$(libdir)/libdmumps$(PLAT)$(LIBEXT):
+$(libdir)/libdmumpso$(PLAT)$(LIBEXT):
 	@echo 'Error: you should build the library' $@ 'first'
 	exit 1
 
-$(libdir)/libcmumps$(PLAT)$(LIBEXT):
+$(libdir)/libcmumpso$(PLAT)$(LIBEXT):
 	@echo 'Error: you should build the library' $@ 'first'
 	exit 1
 
-$(libdir)/libzmumps$(PLAT)$(LIBEXT):
+$(libdir)/libzmumpso$(PLAT)$(LIBEXT):
 	@echo 'Error: you should build the library' $@ 'first'
 	exit 1
 
